data/json/dreams.json

```
@@ -232,14 +234,19 @@
     "type": "dream",
     "messages": [
       "You have a vivid dream of hunting in the woods.",
-      "Whilst dreaming, you see a disturbingly bestial version of yourself."
+      "Whilst dreaming, you see a disturbingly bestial version of yourself.",
+      "You dream of blood and guts. For some reason, it makes you feel weirdly excited."
     ],
     "category": "BEAST",
     "strength": 2
   },
   {
     "type": "dream",
-    "messages": [ "You dream of foraging in the woods… mouth-first?", "Your dream-reflection is rather bearish." ],
+    "messages": [
+      "You dream of foraging in the woods… mouth-first?",
+      "Your dream-reflection is rather bearish.",
+      "You dream of brushing the fur on your back against a tree. Wait, fur?"
```

```suggestion
      "You dream of brushing the fur on your back against a tree."
```
There's a good chance the character already has fur, and if they don't, it is more dreamlike this way